app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -20,72 +20,106 @@
-            id="ingredient-unit"
-            v-model="form.measure"
-          >
-            <!--Add Mode unit Unselected -->
-            <option
-              v-if="!editMode"
-              hidden
-              selected
+        <div
+          v-if="index == 1"
+        >
+          {{ $t('msg.ingredients.alternative_unit') }}
+        </div>
+        <div class="flex flex-wrap -mx-3">
+          <div class="w-full md:w-1/2 px-3 mb-6 md:mb-0">
+            <!--Quantity -->
+            <label
+              v-if="index == 0 || index == 1"
+              class="block text-gray-700 text-sm font-bold mb-2"
+              for="ingredient-quantity"
             >
-              {{ $t('msg.ingredients.measure') }}
-            </option>
-            <!--Edit Mode unit ingredient -->
-            <option
-              v-if="editMode"
-              selected
-              :key="ingredient.measure"
-              :value="ingredient.measure"
+              {{ $t('msg.ingredients.quantity') }}
+            </label>
+            <input
+              class="appearance-none block w-full bg-white text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none"
+              v-model="unit.quantity"
+              type="number"
+              :placeholder="$t('msg.ingredients.quantity')"
             >
-              {{ ingredient.measure }}
-            </option>
-            <!--Other units -->
-            <option
-              v-for="unit in formUnits"
-              :key="unit"
-              :value="unit"
+          </div>
+          <div class="relative">
+            <!--Measure -->
+            <label
+              v-if="index == 0 || index == 1"
+              class="block text-gray-700 text-sm font-bold mb-2"
+              for="ingredient-unit"
             >
-              {{ unit }}
-            </option>
-          </select>
+              {{ $t('msg.ingredients.measure') }}
+            </label>
+            <div class="flex">
+              <select
+                class="block appearance-none w-full bg-white border border-gray-200 text-gray-700 py-3 px-4 pr-8
+                rounded leading-tight focus:outline-none"
+                id="ingredient-unit"
+                v-model="unit.name"
+              >
+                <!--Add Mode unit Unselected -->
+                <option
+                  v-if="!editMode"
+                  hidden
+                  selected
+                >
+                  {{ $t('msg.ingredients.measure') }}
+                </option>
+                <!--Edit Mode unit ingredient -->
+                <option
+                  v-if="editMode"
+                  selected
+                  :key="ingredient.measure"
+                  :value="ingredient.measure"
+                >
+                  {{ ingredient.measure }}
+                </option>
+                <!--Other units -->
+                <option
+                  v-for="unit in formUnits"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Variable 'unit' is already declared in the upper scope            vue/no-template-shadow